src/certifier.mak

```diff
@@ -83,61 +83,61 @@ $(CL)/certifier.a: $(dobj)
 	$(AR) rcs $(CL)/certifier.a $(dobj)
 
 $(S)/certifier.pb.cc $(I)/certifier.pb.h: $(CP)/certifier.proto
-	$(PROTO) --cpp_out=$(S) --proto_path $(CP) $<
+	$(PROTO) --cpp_out=$(S) --proto_path $(<D) $<
```

Comment:
The variable `$(<D)` means: use the dir of the 1st pre-requisite which here is $(CP).

This way, we more tightly bind the call to `protoc` to use the directory where the `certifier.proto` file lives as the  value of argument `--proto_path`.